exercises/chapter4/test/Main.purs

```diff
@@ -202,7 +202,7 @@ runChapterExamples =
       Assert.equal 120
         $ fact 5
     test "fib" do
-      Assert.equal 55
+      Assert.equal 34
```

Comment:
Looks like this line was missed by https://github.com/purescript-contrib/purescript-book/pull/351

Reply:
Oh, thanks for diagnosing that one!